zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -10675,29 +10690,8 @@ struct DecodableType
 } // namespace TimeSynchronization
 namespace BridgedDeviceBasicInformation {
 namespace Structs {
-namespace ProductAppearanceStruct {
-enum class Fields : uint8_t
-{
-    kFinish       = 0,
-    kPrimaryColor = 1,
-};
-
-struct Type
-{
-public:
-    ProductFinishEnum finish = static_cast<ProductFinishEnum>(0);
-    DataModel::Nullable<ColorEnum> primaryColor;
-
-    CHIP_ERROR Decode(TLV::TLVReader & reader);
-
-    static constexpr bool kIsFabricScoped = false;
-
-    CHIP_ERROR Encode(TLV::TLVWriter & aWriter, TLV::Tag aTag) const;
-};
-
-using DecodableType = Type;
-
-} // namespace ProductAppearanceStruct
+namespace CapabilityMinimaStruct  = Clusters::detail::Structs::CapabilityMinimaStruct;
+namespace ProductAppearanceStruct = Clusters::detail::Structs::ProductAppearanceStruct;
```

Comment:
Technically, this is a breaking change: if someone was assuming that BasicInformation::ProductAppearanceStruct::Type and BridgedDeviceBasicInformation::ProductAppearanceStruct::Type are different types and wrote code accordingly, they will now lose, in the "your code does not compile" sense, probably.

@hasty we should figure out how we want to represent these things in XML; there are some subtle problems with just attaching cluster IDs to a single decl, especially as a change from what was there before.